chemicaltoolbox/rdock/rbdock.xml

```diff
@@ -0,0 +1,286 @@
+<tool id="rbdock" name="rDock docking" version="0.1">
```

Comment:
Is this tool multi-core enabled? Should GALAXY_SLOTS be used?
If not do we want to parallelize it with gnu-parallel as well?

Reply:
@bgruening rdock only utilises a single core. My expectation was that the input would be split into multiple files (a collection) on the galaxy side and that collection used as input to the tool.
In order to use gnu parallel I think we would need to split the input as part of the execution?

Reply:
@tdudgeon you don't anticipate that this submitting a separate job for each ligand will cause similar problems to those you had with Vina?

Reply:
No, because each rdock execution handles multiple ligands (input is SDF), and you can tune the number to give a sensible balance between execution time and parallelism.